crates/chia-consensus/src/merkle_set.rs

```diff
@@ -147,567 +185,4 @@ fn radix_sort(range: &mut [[u8; 32]], depth: u8) -> ([u8; 32], NodeType) {
     }
 }
 
-pub fn compute_merkle_set_root(leafs: &mut [[u8; 32]]) -> [u8; 32] {
-    // There's a special case for empty sets
-    if leafs.is_empty() {
-        return BLANK;
-    }
-
-    match radix_sort(leafs, 0) {
-        (hash, NodeType::Term) => {
-            // if there's only a single item in the set, we prepend "Term"
-            // and hash it
-            // the reason we don't just check the length of "leafs" is that it
-            // may contain duplicates and boil down to a single node
-            // (effectively), which is a case we need to support
-            let mut hasher = Sha256::new();
-            hasher.update([NodeType::Term as u8]);
-            hasher.update(hash);
-            hasher.finalize().into()
-        }
-        (hash, NodeType::Mid) => hash,
-        (hash, NodeType::MidDbl) => hash,
-        (_, NodeType::Empty) => panic!("unexpected"),
-    }
-}
-
-#[test]
-fn test_get_bit_msb() {
```

Comment:
The tests also test the new code at the same time, so to prevent circular imports and leave your code as intact as possible I moved them to the new file

Reply:
As far as I'm aware it's fine to have two files use items from each other in Rust, as long as they are part of the same crate.

Reply:
Either way the tests have been expanded to test the new things so it makes to put them with the new things. There is a comment at the bottom of merkle_set pointing to where the tests now are

Reply:
it makes it very hard to review what happened to them, when they're moved and change in the same PR.

Reply:
Ok, I've moved the tests back into merkle_set.rs